app/assets/javascripts/application.js

```diff
@@ -13,5 +13,5 @@
 //= require jquery
 //= require rails-ujs
 //= require activestorage
-//= require_tree .
+//= require cable
```

Comment:
can you undo this change?

Reply:
changed to "//= require_tree ./channels" to load successfully the scripts:
```
<script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.5.1/dist/algoliasearch-lite.umd.js"
    integrity="sha256-EXPXz4W6pQgfYY3yTpnDa3OH8/EPn16ciVsPQ/ypsjk=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4.8.3/dist/instantsearch.production.min.js"
    integrity="sha256-LAGhRRdtVoD6RLo2qDQsU2mp+XVSciKRC8XPOBWmofM=" crossorigin="anonymous"></script>
```
with "//= require_tree ." scripts are not load successfully, for that reason @dperezserra  changed to cable, is there a better way to do it?

Reply:
It seems you both are right https://stackoverflow.com/questions/39294859/require-tree-argument-must-be-a-directory-in-a-rails-5-upgraded-app/39312176 change accepted!